src/blocker.rs

```diff
@@ -130,31 +135,34 @@ impl Blocker {
         self.get_list(NetworkFilterListId::TaggedFiltersAll)
     }
 
-    #[cfg(feature = "single-thread")]
-    fn borrow_regex_manager(&self) -> std::cell::RefMut<'_, RegexManager> {
+    pub(crate) fn borrow_regex_manager(&self) -> RegexManagerRef<'_> {
+        #[cfg(feature = "single-thread")]
         #[allow(unused_mut)]
         let mut manager = self.regex_manager.borrow_mut();
+        #[cfg(not(feature = "single-thread"))]
+        let mut manager = self.regex_manager.lock().unwrap();
 
         #[cfg(not(target_arch = "wasm32"))]
         manager.update_time();
 
         manager
     }
 
-    #[cfg(not(feature = "single-thread"))]
-    fn borrow_regex_manager(&self) -> std::sync::MutexGuard<'_, RegexManager> {
-        let mut manager = self.regex_manager.lock().unwrap();
-        manager.update_time();
-        manager
-    }
-
     pub fn check_generic_hide(&self, hostname_request: &Request) -> bool {
         let mut regex_manager = self.borrow_regex_manager();
         self.generic_hide()
             .check(hostname_request, &HashSet::new(), &mut regex_manager)
             .is_some()
     }
 
+    #[doc(hidden)]
+    pub(crate) fn check_exceptions(&self, request: &Request) -> bool {
```

Comment:
no need to include `#[doc(hidden)]` on `pub(crate)` items

Reply:
fixed, thanks